README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# HW 01
+
+### Introduction
+
+![-](https://user-images.githubusercontent.com/67297411/85501690-7d090680-b5ab-11ea-9c45-1364676d9eda.png)
+
+Hello! My name is Pranathi Posa, and I'm a rising third-year in The College. I'm majoring in Public Policy and minoring in Statistics. I write and edit for *The Maroon* and *Polytics*, and I'm also a part of ChoMUN.
+
+### Here are Some of My Favorite...
+
+**Books?** *Good Omens* by Neil Gaiman, *The Martian* by Andy Weir, *Extremely Loud & Incredibly Close* by Jonathan Safran Foer
+
+**Movies?** Spider-Man: Into the Spiderverse, The Man from U.N.C.L.E, What We Do in the Shadows
+
+**TV Shows?** The Witcher, Avatar: The Last Airbender, Broadchurch
+
+**Vines?** [Juanita Loves Jesus](https://www.youtube.com/watch?v=Rsk8HsAxTEI), [Fed Up Teacher](https://www.youtube.com/watch?v=dQw4w9WgXcQ), [Cat Horn](https://www.youtube.com/watch?v=XMIXArF5sSU)
+
+**Albums?** *Imagination & the Misfit Kid* by Labrinth, *American Beauty / American Psycho* by Fall Out Boy, *Dear Wormwood* by The Oh Hellos, *Melodrama* by Lorde
+
+<font size="1"> Yes, I made the Vine section just to make that joke. No, I don't regret it. </font>
```

Comment:
Okay so here's the deal. I get what you were trying to do but it won't work. Markdown (especially on GitHub) is especially spartan. On the scale of word processors from Notepad++/TextEdit all the way to MS Word, Markdown exists here:
```
Notepad++/TextEdit–––––––Markdown–––––––––––––––––––––––––––––––––––– MS Word
```
That means you don't have as much customizability as you would expect otherwise. Resizing images, text, or anything else is difficult in markdown as result. Great attempt though!